### ISSUE TITLE
Drupal support

### ISSUE THREAD

**Issue by warmth** [blasten/turn.js#151]
Hey Emmanuel, how about porting turn.js to Drupal?

http://drupal.org/node/1760282

**Comment by ryHollingsworth** [comment]
someone has taken v3 and made a wordpress plugin.. drupal and joomla plugins are a matter of time :)
http://codecanyon.net/item/responsive-flip-book-powered-by-jquery/1724917

**Comment by blasten** [comment]
@warmth @ryHollingsworth
Yes, I think that would be so cool. Are one of you a Drupal or Joomla expert?

**Comment by nicholasgame** [comment]
I've used v3 in a Drupal 7 site and it works well, it can bug out from time to time but it can be done.
http://dftv2012.co.uk/

**Comment by warmth** [comment]
Any news about this? The dev turned down his intentions to implement it due the lack of time...

**Comment by blasten** [comment]
@warmth do you know that developer?  We could work something out together.

**Comment by warmth** [comment]
Yes, just write in that thread (look on my first post) maybe that could make this happen.
